Tests/fixtures/tv-1.yml

```diff
@@ -4091,3 +4091,1000 @@
     model: X4S
   os_family: Android
   browser_family: Chrome
+-
+  user_agent: Opera/9.80 (Linux mips; Opera TV/39) Presto/2.11.355 Version/12.11
+  os:
+    name: GNU/Linux
+    version: ""
+    platform: MIPS
```

Comment:
(optional)
here we can define OS - Opera TV
https://blogs.opera.com/news/2016/01/ces-2016-meet-opera-tv/

regex: `Linux (?:[:^;]+); Opera TV(?: Store)?/`
os_family GNU/Linux

Opera/9.80 (Linux mips; Opera TV/4) Presto/2.11.355 Version/12.11
Opera/9.80 (Linux armv7l; Opera TV Store/5581) Presto/2.12.362 Version/12.11